image/rhel/Dockerfile.envsubst

```diff
@@ -55,7 +55,7 @@ RUN ln -s entrypoint-wrapper.sh /stackrox/admission-control && \
     microdnf clean all && \
     rm /tmp/snappy.rpm /tmp/postgres.rpm /tmp/postgres-libs.rpm RPM-GPG-KEY-CentOS-Official && \
     # (Optional) Remove line below to keep package management utilities
-    rpm -e --nodeps $(rpm -qa curl '*rpm*' '*dnf*' '*libsolv*' '*hawkey*' 'yum*') && \
```

Comment:
Why are we removing this here? (I am not sure what this file does, that is why I am asking)

Reply:
This is the spot where we remove all package managers from the main image. 
I have only removed `*RPM*` from this line, as we want to keep RPM in the images for the `rpmdb` command Compliance needs.
IIUC, this file is the template from which the Dockerfile for `main` is generated.

Reply:
Oh, now I recall. We remove it from the removal code, so in fact we are keeping the package. All good

Reply:
Due to a lack of a follow-up `// TODO(ROX-12345): comment`, I have a question: where do we stand in this decision? Related Slack thread looks unfinished to me https://srox.slack.com/archives/C04213W844U/p1670426415333499